pkgs/tools/misc/esphome/dashboard.nix

```diff
@@ -4,13 +4,15 @@
 
 with python3.pkgs; buildPythonPackage rec {
   pname = "esphome-dashboard";
-  version = "20210615.0";
+  version = "20210617.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "07ammr46bipfi4b7nnjkip5l7966wxqhp5n3g2wqf68m3ymx24s9";
+    sha256 = "0kwg940pdzjsfbdmcml382khpxm6p1ip0bib0wk8rg8qdvmia7w5";
   };
 
+  doCheck = false;
```

Comment:
Please add a comment why tests got disabled. Also we probably want to add a pythonImportsCheck.